### ISSUE TITLE
Please specify a license for this repo

### ISSUE THREAD

**Issue by 54696d21** [mozilla/DeepSpeech-examples#117]

**Comment by lissyx** [comment]
Please explain why, and also understand it's a repo containing code from multiple authors ...

**Comment by 54696d21** [comment]
I wrote something small based on the vad_transcriber and want to know if I can publish it and if so under what license

**Comment by lissyx** [comment]
@54696d21 You should ping the person who wrote it?

**Comment by lissyx** [comment]
@bak What is your license for this code ?

**Comment by lissyx** [comment]
@54696d21 If you are willing to help us and reach out to authors to clarify the license were needed, feel free: at first, since it was hosted in the main repo, I think people were okay and it should fall under the MPL, but clarification can be welcome ?

**Comment by lissyx** [comment]
FTR, latest examples on the old repo before moving to this one: https://github.com/mozilla/DeepSpeech/tree/v0.6.0/examples

**Comment by 54696d21** [comment]
Ok, if the code I took originally appeared in that repo which is under MPL then I should be ok. Thank you very much :)

**Comment by reuben** [comment]
Ouch. This was a stupid oversight when moving the examples from the main repo to here :(

The intention has always been to keep MPL-2.0 like the main repo, but now we have to reach out to people who contributed in the mean time to confirm they're OK with the license. I'll do that in this issue.

**Comment by Andrew-Chen-Wang** [comment]
@reuben If the original repo's code was simply migrated without modification, then I don't see a problem with slapping the MPL-2.0 license on to this repository? I'm no lawyer, <sub>this shouldn't be legal advice, Mozilla should have more than enough lawyers,</sub> and I actually haven't fully read the MPL-2.0 license before, but assuming it's similar to the other modern open source license, it's fair game to modify the code however you wish without notifying anyone and thus the code should be compatible with any new license it's fallen under, even if it's the same from the original repo... at least I think.

**Comment by reuben** [comment]
> @reuben If the original repo's code was simply migrated without modification, then I don't see a problem with slapping the MPL-2.0 license on to this repository?

Correct. Unmodified code from the previous repo is still MPL-2.0. The author consent is needed for modifications contributed directly here, after the repo move.

**Comment by Andrew-Chen-Wang** [comment]
Ah I didn't realize there were some minor contributions made here as well. Duly noted!

**Comment by chriswells0** [comment]
This is sort of a big deal, right? The more changes are merged, the harder it will get for people.

I'm in the same spot: I'd like to share some small changes to _mic_vad_streaming.py_ that adds basic recognition of a few voice commands while transcribing speech to text. It looks like that file would fall under the MPL, but I'm also not a lawyer and don't feel comfortable making a guess at whether it's legal to share. Maybe I could refactor to have a script that calls _mic_vad_streaming.py_ instead...
